mne/io/edf/_utils.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""Helper functions for EDF, EDF+, BDF converters to FIF."""
+
+# Authors: Teon Brooks <teon.brooks@gmail.com>
+#          Martin Billinger <martin.billinger@tugraz.at>
+#          Nicolas Barascud <nicolas.barascud@ens.fr>
+#          Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+#          Joan Massich <mailsik@gmail.com>
+#
+# License: BSD (3-clause)
+
+import re
+from ...utils import hashfunc
+
+
+def _load_gdf_events_lut(fname, md5):
+    assert hashfunc(fname, hash_type='md5') == md5
```

Comment:
My usual complaint, since this is not a test file I would use try/except instead of `assert`.